### ISSUE TITLE
LINK1181 fatal error:Release\image_lib.lib"  can not be opened

### ISSUE THREAD

**Issue by bonejay** [gabrieleilertsen/briefmatch#1]
"C:\briefmatch\build\ALL_BUILD.vcxproj" (Standardziel) (1) ->
"C:\briefmatch\build\briefmatch.vcxproj" (Standardziel) (3) ->
(Link Ziel) ->
  LINK : fatal error LNK1181: Eingabedatei "Release\image_lib.lib" kann nicht ge÷ffnet werden. [C:\briefmatch\build\briefmatch.vcxproj]

Under win10 using visual studio 2017

**Comment by gabrieleilertsen** [comment]
Difficult to say what the problem is without further information. Has the image_lib library been built during compile time, but cannot be opened to link against? I have not tested myself to compile under Windows, but I don't see why it shouldn't work.

**Comment by bonejay** [comment]
Here is my image_lib folder where there is only the .dll file.
![briefmatch](https://user-images.githubusercontent.com/31944379/41309049-e50fbcb4-6e7d-11e8-82c1-916c6c3a7a28.png)

**Comment by gabrieleilertsen** [comment]
The problem seems to be that a dynamic library is created (image_lib.dll), but the linker is looking for the static library (image_lib.lib). I am not very familiar with Visual Studio, but you don't directly link against dynamic libraries (see e.g. [this](https://stackoverflow.com/questions/7845886/linking-dll-in-visual-studio) thread).

Alternatively, I guess you can assure that the static library is built instead of the dynamic. There should be [options](https://stackoverflow.com/questions/5040965/how-to-switch-a-project-built-on-visual-studio-2010-from-dynamic-to-static) in Visual Studio to change this. Otherwise, you can change this in the CMake before creating the solution, by changing SHARED to STATIC on [the appropriate row](https://github.com/gabrieleilertsen/briefmatch/blob/eb981fa876a4df43c915f85686ba13364d8922f7/CMakeLists.txt#L28).

**Comment by bonejay** [comment]
I changed the cmake file to your suggestion and this came out. Only more conflicts 
with dynamic libraries.
![briefmatch2](https://user-images.githubusercontent.com/31944379/41491399-2e162a9c-70f9-11e8-8e69-67ab424213f5.png)

**Comment by gabrieleilertsen** [comment]
Yes, I guess you would have to provide static builds of all the included libraries, so maybe it only makes it more difficult. It should be possible to use the dynamic build of imageLib in some way, but I have to little experience with VS to point to how to do this. Currently, I have no Windows machine to test myself, but I will update the code and/or build instructions when this is the case.
